samples/remote_attestation/common/dispatcher.cpp

```diff
@@ -195,36 +362,66 @@ int ecall_dispatcher::generate_encrypted_message(uint8_t** data, size_t* size)
     }
 
     encrypted_data_size = sizeof(encrypted_data_buf);
-    if (m_crypto->Encrypt(
-            m_crypto->get_the_other_enclave_public_key(),
+    memset(encrypted_data_buf, 0x00, encrypted_data_size);
+    if (m_crypto->encrypt_gcm(
+            m_enclave_config->sym_key,
+            ++(m_enclave_config->sequence_number),
             m_enclave_config->enclave_secret_data,
             ENCLAVE_SECRET_DATA_SIZE,
+            iv_str,
             encrypted_data_buf,
-            &encrypted_data_size))
+            &encrypted_data_size,
+            tag_str))
     {
-        uint8_t* host_buf = (uint8_t*)oe_host_malloc(encrypted_data_size);
+        // Reason we allocate memory in the host is so that the encrypted data
```

Comment:
At least fix grammar...
"We allocate memory in the host so that..."
And add a TODO to fix this.

Reply:
Will rephrase.

Reply:
Rephrased and also filed an issue to deprecate oe_host_malloc in tests and samples